<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawalView</class>
 <widget class="QWidget" name="WithdrawalView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <property name="bottomMargin">
    <number>50</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelInfo">
     <property name="text">
      <string>Select the amount of withdrawal</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnWithdraw20">
     <property name="text">
      <string>20€</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnWithdraw100">
     <property name="text">
      <string>100€</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnWithdraw50">
     <property name="text">
      <string>50€</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnWithdraw200">
     <property name="text">
      <string>200€</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnWithdrawX">
     <property name="text">
      <string>Other</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnWithdraw500">
     <property name="text">
      <string>500€</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
